doc/decisions/manpages.md

```diff
@@ -11,17 +12,23 @@ Storing generated files is annoying, as it requires:
 
 ## Constraints
 
-1. we want beautiful rendered man pages
-2. we cannot require rare tools for the build process
+1. we want beautiful rendered manpages, e.g., OPTIONS section looks like normal manpages, see in Notes¹ below
+2. we cannot require rare tools for the build process: the manpages must be present in every package
 
 ## Assumptions
 
-- ronn-ng doesn't have packages on any distribution (violates constraint 2.)
-
 ## Considered Alternatives
 
-1. Doxygen: need to be rechecked in the latest version, see [#4551](https://issues.libelektra.org/4551)
-2. Pandoc 2.9 doesn't have the Markdown description-list feature like ronn-ng has (violates constraint 1.)
+0. `ronn-ng`, which doesn't have packages on most distribution (violates constraint 2.) and thus created this problem
+1. Write a tool that converts our specification, similar to [pythongen](/src/tools/pythongen/template/template.man)
+2. Write a tool that parses gopts `--help` output
+3. [help2man](https://www.gnu.org/software/help2man/)
+4. Doxygen:
+   - Constraint 1 probably broken
+5. Pandoc: has quite a few dependencies and would need rewrite of the current documentation in doc/help:
```

Comment:
The required dependencies [listed for Debian](https://packages.debian.org/bullseye/pandoc) all seem like pretty standard packages. The dependencies of the RPM package also seem very similar.